tianshou/data/buffer/base.py

```diff
@@ -342,6 +398,10 @@ def get(
         index: int | list[int] | np.ndarray,
         key: str,
         default_value: Any = None,
+        # TODO 1: this is only here because of atari, it should never be needed (can be solved with index)
+        #  and should be removed
+        # TODO 2: does something entirely different from getitem
+        # TODO 3: key should not be required
         stack_num: int | None = None,
```

Comment:
`stack_num` can potentially be used for RNNs. I think I saw this note also in one of the tutorials. But if there is no big support planned for RNNs on the horizon, it can probably be removed.